.github/workflows/ui-install-and-lint.yml

```diff
@@ -0,0 +1,70 @@
+name: FOLIO UI Install Packages
+
+on:
+  workflow_call:
+    inputs:
+      # yarn configuration
+      node-version:
+        description: Node version to use, in semver notation
+        required: true
+        type: string
+      folio-npm-registry:
+        description: Full URL to the registry to use for @folio modules
+        required: true
+        type: string
+      debug-yarn-packages:
+        description: Whether to print all packages installed by yarn. Defaults to false
+        required: false
+        type: boolean
+        default: false
```

Comment:
Can we configure this so it operates as "always publish on fail?" e.g. by chaining it to the output status of the "test" command? We know we'll want `yarn.lock` if we fail, so we should just provide it.

Reply:
sure; we could also just always have it publish with a small (1 day?) expiry